doubtlab/reason.py

```diff
@@ -72,6 +72,44 @@ def __call__(self, X, y=None):
         return np.where(rvals < self.probability, rvals, 0)
 
 
+class ShannonEntropyReason:
+    """
+    Assign doubt when the normalized Shannon entropy is too high.
```

Comment:
Maybe add a link to the definition of normalised Shannon entropy?  I'm not 100% sure if the docs are configured to render mathjax, but in the meantime we could link to [here](https://math.stackexchange.com/questions/395121/how-entropy-scales-with-sample-size).

Reply:
Aye!